src/components/ExploreNav/styled.js

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components";
+import { BLACK, H3, CREAM, PINK } from "../../styles/styles";
+import { min } from "../../styles/breakpoints";
+import Cursor from "../../images/cursor-pointer.svg";
+import { AnchorLink } from "gatsby-plugin-anchor-links";
+
+export const StyledLink = styled.div`
+  white-space: nowrap;
+  height: 3rem;
+  width: 100%;
+  transform: skew(15deg);
+  background-color: transparent;
+  cursor: url(${Cursor}) 20 20, pointer;
+  border: 2px solid ${CREAM};
+  text-align: center;
+  &:hover {
+    background-color: ${PINK};
+  }
+`;
+
+export const LeftLink = styled(StyledLink)`
+  border-right: none;
+`;
+
+export const CenterLink = styled(StyledLink)`
+  border-right: none;
+  border-left: none;
+`;
+
+export const RightLink = styled(StyledLink)`
+  border-left: none;
+`;
+
+export const LinkContainer = styled.div`
```

Comment:
can you make this sticky? I believe the designers intended the bar to be constant at the top as you scroll